frontend/src/app/components/add-post-modal/add-post.component.ts

```diff
@@ -32,6 +39,8 @@ export interface AddPostDialog {
   styleUrls: ['./add-post.component.scss'],
 })
 export class AddPostComponent {
+  @ViewChild('titleInput', { static: true })
+  titleInputElement!: ElementRef<HTMLInputElement>;
```

Comment:
Can we also remove this since it was used by the commented function which I don't think we need anymore.